### PR TITLE
https://trello.com/c/YNnHScUp Loading the job config from the Google drive

### PR DESCRIPTION
**Pull request by gisly**
https://trello.com/c/YNnHScUp

Loading the job config from the Google drive

### CODE REVIEW

File: src/tg/handlers/access_config_handler.py [hunk 1]
```diff
@@ -49,6 +49,24 @@ def get_config_jobs(update, tg_context):
     )
 
 
+@admin_only
+def reload_config_jobs(update, tg_context):
+    try:
+        jobs_config_file_key = ConfigManager().get_jobs_config_file_key()
+        if jobs_config_file_key is None:
+            raise Exception("No jobs config file key provided")
+        jobs_config_json = AppContext().drive_client.download_file(jobs_config_file_key)
```
Comment (alexeyqu) [inline src/tg/handlers/access_config_handler.py:58]: we return `bytes` but parse as json, hmm

File: src/drive/drive_client.py [hunk 2]
```diff
@@ -77,6 +78,19 @@ def is_open_for_edit(self, file_url: str) -> bool:
...
         return False
 
+    def download_file(self, file_id: str) -> bytes:
+        try:
+            request = self.service.files().get_media(fileId=file_id)
+            file = io.BytesIO()
+            downloader = MediaIoBaseDownload(file, request)
+            done = False
+            while done is False:
+                status, done = downloader.next_chunk()
```
Comment (alexeyqu) [inline src/drive/drive_client.py:88]: ouch, looks complicated.. Is `MediaIoBaseDownload` the best class to use?

**Review by alexeyqu — CHANGES_REQUESTED** [review]
Can you add some words to PR title (not only the Trello URL)?
Have a look at some questions on the Drive API/client usage.

File: src/tg/handlers/access_config_handler.py (hunk 1 above)
Comment (gisly) [reply src/tg/handlers/access_config_handler.py:58]: https://docs.python.org/3/library/json.html#json.loads loads seems to work well with bytes but I can parse right here to make it more readable

**Review by alexeyqu — APPROVED** [review]

**Pull request merged**